CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 - General
+  - Fix unit tests on MS-Windows. #1211
```

Comment:
I think we could improve this mentioning what affects the final user, like "Improve MS-Windows support fixing path issues and etc"

Reply:
Done